superset/assets/src/explore/components/controls/SubscriberLayer.jsx

```diff
@@ -341,10 +351,21 @@ export default class SubscriberLayer extends React.PureComponent {
                 onChange={v => this.setState({ name: v })}
                 validationErrors={!name ? [t('Mandatory')] : []}
               />
+
+              <CheckboxControl
+              hovered
+                name="annotation-layer-hide-line"
+                label="Use as popup"
+                description={'If selected, this slice will not available to use in dashboard.'}
```

Comment:
Description is not clear to me.  As per this description, **we are not able to use in dashboard** give wrong msg to end user.

Reply:
Use this option to add this slice in dashboard as a Modal.

Reply:
done